### PR TITLE
Add preliminary acquisuite POST data collection.

### PR DESCRIPTION
**Pull request by jackrwoods**
This adds a new route for acquisuite POST data collection, and saves the body of the request in a file. I have also added the new acquisuite data folder to the .gitignore file, for the time being.

### CODE REVIEW

File: Source/server.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ var port     = process.env.PORT || 3000;
 var mongoose = require('mongoose');
 var passport = require('passport');
 var flash    = require('connect-flash');
+var fs       = require('fs'); // TEMP - for saving acquisuite POST data
```
Comment (DSchroederOSU) [inline Source/server.js]: what is fs? node module?

File: Source/app/routes.js [hunk 2]
```diff
@@ -72,6 +72,26 @@ module.exports = function(app, passport) {
         })
     );
 
+    //=====================================
+    // Data Collection
+    //=====================================
+    // Collects POST data from acquisuites,
+    // converts it to JSON, and saves it to
+    // the database.
+    app.post('/acquisuite/upload/:id', function (req, res) {
```
Comment (DSchroederOSU) [inline Source/app/routes.js]: will req be coming from web socket server/db server?

File: Source/server.js (hunk 1 above)
Comment (jackrwoods) [reply Source/server.js]: FS is a node module called "File System." It allows a node app to create/edit/delete files locally on the server. In this case, fs will create a file called "postData.txt" that'll store the contents of the POST data. Once we get some sample data and set up the database, fs will no longer be needed.

File: Source/app/routes.js (hunk 2 above)
Comment (jackrwoods) [reply Source/app/routes.js]: As far as I know, the request will come from the web server. I'm using Express's built in req object ([more on that HERE](http://expressjs.com/en/api.html#req)).

Comment (jackrwoods) [reply Source/app/routes.js]: We used this method to collect POST data in CS 290, anyway. It was pretty straightforward.

Comment (DSchroederOSU) [reply Source/app/routes.js]: Right. But the $HTTP.posts will probably becoming from a web socket server or DB server that we set up, isn't that what we agreed upon?

File: Source/server.js (hunk 1 above)
Comment (DSchroederOSU) [reply Source/server.js]: Beautiful

File: Source/app/routes.js (hunk 2 above)
Comment (DSchroederOSU) [reply Source/app/routes.js]: We use the same method to receive posts from the Angular services as well. The implementation is correct, just curious if you had a plan for where that post was coming from.

Comment (jackrwoods) [reply Source/app/routes.js]: Are you available to hop on Hangouts for a sec?

Comment (jackrwoods) [reply Source/app/routes.js]: It's no big deal but I might have misunderstood how we were structuring the data collection.

**Pull request merged**